src/types/LiftTypes.tsx

```diff
@@ -2,17 +2,50 @@ type Set = {
   reps: number;
 };
 
+type LiftInfoLink = {
+  text: string;
+  url: string;
+};
+
+type LiftInfo = {
+  comment: string;
+  links: LiftInfoLink[];
+};
+
 type LiftLogEntry = {
   date: Date;
   name: string;
   weightLifted: number;
   sets: Set[];
+} & LiftInfo;
```

Comment:
At first glance this kind of type composition seems really convenient.

Reply:
hm... how would you do it differently? `LiftInfo` has to be used in two places - the main log entry type and the add-reps-dialog component's type.

Reply:
Yeah, that's why I thought it was convenient.